config.sh

```diff
@@ -4,34 +4,24 @@
 source gfortran-install/gfortran_utils.sh
 
 function build_wheel {
+    export FFLAGS="$FFLAGS -fPIC"
     if [ -z "$IS_OSX" ]; then
-        unset FFLAGS
-        export LDFLAGS="-shared -Wl,-strip-all"
```

Comment:
Just checking: this can be removed because it's done as a post-processing step by multibuild (https://github.com/matthew-brett/multibuild/blob/cdffc52846355707f389808414cf8dfe31289328/configure_build.sh#L44)? I'm not quite sure why in that code macOS is special-cased and doesn't have `STRIP_FLAGS`.

Reply:
Previously, because of numpy's LDFLAGS behaviour (overwriting instead of appending), this was needed, but not anymore.
`-shared` is not needed anymore due to that and strip flags are set by multibuild anyway.